src/clojure/nrepl/util/classloader.clj

```diff
@@ -0,0 +1,27 @@
+(ns nrepl.util.classloader
```

Comment:
You might mention in the new namespaces that they were introduced in nREPL (`:added`).